src/reporting/ReportingWindow.sol

```diff
@@ -289,4 +291,22 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable {
         numberOfReportsByMarket[_market] += 1;
         return true;
     }
+
+    function getWinningPayoutDistributionHashFromFork(Market _market) constant returns (bytes32) {
+        if (getBranch().getForkingMarket() != _market) {
+            return 0;
+        }
+        ReputationToken _winningDestination = getReputationToken().getTopMigrationDestination();
+        if (address(_winningDestination) == address(0)) {
+            return 0;
+        }
+        if (_winningDestination.totalSupply() < 11 * 10**6 * 10**18 / 2 && block.timestamp < getBranch().getForkEndTime()) {
```

Comment:
Note: There is a good chance that a lot of the comments I have are actually about the original code (which I wrote).  Feel free to ignore such comments until some time in the future, though I'm a fan of the campsite rule (which, when I Google it I only find [this](https://en.wikipedia.org/wiki/Savage_Love#Campsite_rule)... which is not what I had in mind but seems to apply well none the less).